api/edge_api/identities/events.py

```diff
@@ -0,0 +1,19 @@
+import json
+
+import boto3
+from django.conf import settings
+
+events_client = None
+if settings.IDENTITY_MIGRATION_EVENT_BUS:
+    events_client = boto3.client("events")
+
+
+def send_migration_event(project_id: int):
+    event = {
+        "EventBusName": settings.IDENTITY_MIGRATION_EVENT_BUS,
+        "Source": "flagsmith.api.migrate",
```

Comment:
How is this used?

Reply:
It is used in combination with Detail -type to decided if it matches a given (event bridge)rule.

```
source

    Identifies the service that sourced the event. All events sourced from within AWS begin with "AWS." Customer-generated events can have any value here, as long as it doesn't begin with "AWS." We recommend the use of Java package-name style reverse domain-name strings.
 For example, the source value for Amazon CloudFront is aws.cloudfront.
```


ref: https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/CloudWatchEventsandEventPatterns.html